<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="FU-CORP002469" failures="1" tests="2" name="Test" time="33.382" errors="0" timestamp="2023-07-19T16:45:26 IST">
  <testcase classname="com.learnautomation.testcases.DataDrivenTest" name="loginDDT" time="10.682"/>
  <testcase classname="com.learnautomation.testcases.DataDrivenTest" name="loginDDT" time="5.227">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.learnautomation.testcases.DataDrivenTest.loginDDT(DataDrivenTest.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- Test -->
